### ISSUE TITLE
`More` button in panel chat should be disabled when there's no input

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#243595]
1. Open panel chat with no input
2. Navigate via keyboard to the more button and hit enter
3. As a screen reader user, this is confusing because arrow keys do nothing. 

![Image](https://github.com/user-attachments/assets/e57d568a-bc9f-4600-99bf-31d5e3c5ea2e)

**Comment by roblourens** [comment]
Makes sense, I think `DropdownWithPrimaryActionViewItem` should be able to do this for all of its users
